src/test/scala/TypeCheckerSpec.scala

```diff
@@ -203,12 +184,10 @@ class TypeCheckerSpec extends FunSpec {
       }
     }
     it("adding static ints does NOT perform type level computation") {
-      val e1 = typeCheck("let x = 1; let y = 2; let z = x + y;")
-      assert(e1("z") === TSizedInt(2))
+      typeCheck("let x = 1; let y = 2; let z = x + y;")
```

Comment:
Stating the obvious here, but this test (like the ones below) is now a little weaker: it doesn't check the deduced type for `z`.